pages/about.mdx

```diff
@@ -79,17 +104,6 @@ Some “relative” colors use these colors for various components:
 
 Our dark mode is powered by [`prefers-color-scheme: dark`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme).
 
-## Custom domains
```

Comment:
Removed the custom domains block here too.

Reply:
ooo this is actually now back to being available! `_Sadly, we have ran out of space and can't register any more domains._` should probably be deleted

Reply:
EXCITING, i'll add it back in without that note!

Reply:
YAY!

Reply:
@sampoder hows this? I assume the Vercel limitation still stands [69568c3](https://github.com/hackclub/scrapbook/pull/282/commits/69568c3a74ecfe311730fade9326f916094e08f1)

Reply:
Vercel's limitation is also gone now as they took away their limits 🚀